frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -1,5 +1,15 @@
 $tooltip-background-color: #151515;
 
+dl.inline {
```

Comment:
I'd suggest a `co-` prefix so we never conflict with pf or another dependency.

We should either move this to a file like `_common.scss` or give it a BEM name for the monitoring view.

Reply:
Thanks @spadgett. Now renamed to `co-inline` and moved to `_common.scss`.